flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
Do you still need to limit target arches?

Reply:
Yes, dmd is only available on x86 and x86_64